frontend/src/metabase/dashboard/components/DashCard/Dashcard.unit.spec.tsx

```diff
@@ -78,6 +78,12 @@ describe("DashCard", () => {
     expect(screen.getByText("My Card")).toBeVisible();
   });
 
+  it("should not display the ellipsis menu for (unsaved) xray dashboards (metabase#33637)", async () => {
```

Comment:
I put the unit test here as it's the component closest to the menu I found that was already setup with mocks etc